react-to-do-list/src/App.js

```diff
@@ -1,7 +1,12 @@
-import React from 'react'
-
-export default function App() {
-  return (
-    <div>App</div>
-  )
+import React, { Component } from 'react'
+import Home from './pages/home'
+import Poupup from './Component/poupup/poupup'
+export default class App extends Component {
+  render() {
+    return (
+      <div class="container">
+<Home />
```

Comment:
home should be under div level
example: 
<div>
        <Home/>
</div>
you can use **Prettier** to automatic format when save